MetaMask/metamask-extension
# Fixing dark mode in storybook docs view

## **Description**
This PR updates the Storybook configuration to enable Dark Mode in the Docs view, which was previously non-functional. It also removes unnecessary files from the codebase for better code maintainability.

## **Related issues**

Fixes: #19671 

## **Manual testing steps**

1. Go to the storybook build of this PR
2. Turn dark mode on and off buy pressing the 🌞 / 🌙 icon in the top toolbar
3. See that dark mode works for docs pages

## **Screenshots/Recordings**

### **Before**


https://github.com/MetaMask/metamask-extension/assets/8112138/10de57f4-c527-423b-a1b6-8b96f568b895


### **After**


https://github.com/MetaMask/metamask-extension/assets/8112138/5bded120-70ea-44fa-8791-6b7f2ec352c9



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
